### PR TITLE
Add full_test sample information and included description for full-test and other tests 

### PR DESCRIPTION
**Pull request by cying111**
- add samplesheet_full
- update readme

### CODE REVIEW

File: samplesheet_full.csv [hunk 1]
```diff
@@ -0,0 +1,7 @@
+group,replicate,barcode,input_file,genome,transcriptome
+A549,1,,https://dl.dropboxusercontent.com/sh/q098af3xdzfqc72/AAD7YfS6uZNe5wz4Yuam46Dna/SGNex_A549_cDNA_replicate1_run2.fastq.gz,GRCh38,
```
Comment (drpatelh) [inline samplesheet_full.csv:2]:
```suggestion
A549,1,,https://dl.dropboxusercontent.com/sh/q098af3xdzfqc72/AAD7YfS6uZNe5wz4Yuam46Dna/SGNex_A549_cDNA_replicate1_run2.fastq.gz,GRCh37,

```

Comment (drpatelh) [reply samplesheet_full.csv:2]: Maybe we should change this to `GRCh37` because `GRCh38` is currently an NCBI reference....

**Review by drpatelh — COMMENTED** [review]
I think we should change the reference genome version we use in this test @cying111

File: README.md [hunk 2]
```diff
@@ -45,6 +45,42 @@ The pre-basecalled and demultiplexed data in this repository will be used to tes
...
+
+### Files
+
+* `samplesheet_nobc_nodx_noaln.csv` - Sample information sheet required for the pipeline
+* `bam/` - Bam files obtained from [Pre-basecalled and nondemultiplexed data](#pre-basecalled-and-nondemultiplexed-data)
+
+
+## Full-sized test data
+
+The full sized test data in this repository will be used to test the pipeline without the basecalling and demultiplexing step from the SG-NEx Project public dataset. The associated parameters and settings to run the pipeline can be found in [`test_full.config`](https://github.com/nf-core/nanoseq/blob/master/conf/test_full.config).
```
Comment (drpatelh) [inline README.md:77]: Some more detail and maybe links to the project would be nice, and more information about the samples if present.

File: samplesheet_full.csv (hunk 1 above)
Comment (cying111) [reply samplesheet_full.csv:2]: Ok!

File: README.md (hunk 2 above)
Comment (cying111) [reply README.md:77]: Ok! Will do that!

**Review by drpatelh — APPROVED** [review]
Thank you!

**Pull request merged**